app/views/questions/triage/_gyr_tile.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="marketing-comparison">
+  <h2><%= t('.title') %></h2>
+  <p><%= t('.description') %></p>
+  <div class="timeframe">
+    <%= image_tag("clock.svg") %><p><b><%= t('.timeframe') %></b></p>
+  </div>
+  <div class="bottom-section">
+    <div>
+      <div><%= t('questions.triage.collect_your') %></div>
+      <ul class="list--bulleted">
+        <li><%= t('questions.triage.collectables.missing_stimulus') %></li>
+        <li><%= t('questions.triage.collectables.ctc') %></li>
+        <li><%= t('questions.triage.collectables.eitc') %></li>
+        <li><%= t('questions.triage.collectables.additional_credits') %></li>
+      </ul>
+    </div>
+    <div class="spacer"></div>
+    <div class="refund-indicator">
+      <div class="spacing-below-10"><%= t('questions.triage.refund') %></div>
+      <div class="symbols"><%= image_tag("icons/money.svg") %> <%= image_tag("icons/money.svg") %> <%= image_tag("icons/money.svg") %></div>
+    </div>
+  </div>
+  <p class="text--red-bold">
+    <%= t('.warning') %>
+  </p>
+  <div class="spacing-above-35">
+    <%= link_to t('.choose_gyr'), next_path, class: "button button--wide text--centered", "data-track-click": click_tracking_event_name %>
```

Comment:
maybe this is just paranoia, but something about splitting this view into a shared partial makes me worried that `next_path` is going to be different depending on where this is used (not that it currently links to the wrong place, just that it could in the future if rendered on a different page). a couple of options are:
- replace `next_path` with something more explicit, like the other tile views
- pass in the path/url from the view that renders the partial (and that view can pass in `next_path`)
- leave it the way it is, it might be totally fine